python/neutron-understack/neutron_understack/nautobot.py

```diff
@@ -34,6 +62,37 @@ def _truncated(message: str | bytes, maxlen=200) -> str:
     return f"{input[:maxlen]}...{len(input) - maxlen} more chars"
 
 
+class NautobotClient:
```

Comment:
What's the goal of the wrapper class of the Nautobot client? Because other than the last call everything is a single line